pool.go

```diff
@@ -224,7 +224,7 @@ func runPool(options Options) error {
 		handler.header.Set("Access-Control-Allow-Origin", options.Pool.AllowOrigin)
 	}
 
-	if err := handler.Register("vipnode_", p, "connect", "disconnect", "ping", "update", "client", "host"); err != nil {
+	if err := handler.Register("vipnode_", p, "connect", "disconnect", "ping", "update", "peer", "client", "host"); err != nil {
```

Comment:
any way we could lift these up into something enumerable by a unit test to prevent this in the future?

Reply:
Yea, I should either pull out this section into a helper that gets exercised, or make an interface to cast into instead of enumerating, like I do here: https://github.com/vipnode/vipnode/blob/master/agent.go#L200

Reply:
Opened an issue: https://github.com/vipnode/vipnode/issues/55